### PR TITLE
New package: erickutcher.httpdownloader version 1.0.6.5

### LINKED ISSUE #178091: [Package Request]: HTTP Downloader
### How can we help?

I would like someone else to build the manifest.

### Please read and ensure the following

- [X] The installer meets the above requirements

### Please provide the following information

Download Page Url: https://github.com/erickutcher/httpdownloader/releases
Publisher: erickutcher
Package Name: HTTP Downloader
Description: HTTP(S) download manager that uses input/output completion ports (IOCP).
Package Version: v1.0.6.5
Installer Url: https://github.com/erickutcher/httpdownloader/releases/download/v1.0.6.5/HTTP_Downloader_64.zip

### PR DESCRIPTION
**Pull request by Dragon1573**
## Checklist for Pull Requests

- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Resolve #178091

## Manifests

- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/178117)

### CODE REVIEW

**Comment by Dragon1573** [comment]
Manually verified in _Windows Sandbox_ instance.

image

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

File: manifests/e/erickutcher/httpdownloader/1.0.6.5/erickutcher.httpdownloader.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Created with komac v2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: erickutcher.httpdownloader
+PackageVersion: 1.0.6.5
+InstallerType: zip
+NestedInstallerType: portable
+NestedInstallerFiles:
+- RelativeFilePath: HTTP_Downloader.exe
+InstallModes:
```
Comment (Trenly) [inline manifests/e/erickutcher/httpdownloader/1.0.6.5/erickutcher.httpdownloader.installer.yaml:10]:
```suggestion
Commands:
- http-downloader
InstallModes:
```

**Review by Trenly — CHANGES_REQUESTED** [review]

File: manifests/e/erickutcher/httpdownloader/1.0.6.5/erickutcher.httpdownloader.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: erickutcher.httpdownloader
+PackageVersion: 1.0.6.5
+InstallerType: zip
+NestedInstallerType: portable
+NestedInstallerFiles:
+- RelativeFilePath: HTTP_Downloader.exe
+Commands:
+- http-downloader
```
Comment (Trenly) [inline manifests/e/erickutcher/httpdownloader/1.0.6.5/erickutcher.httpdownloader.installer.yaml:11]: My mistake on the previous suggestion - I missed that it was nested in a zip and that just adding `commands` wouldn't work 🤦 
```suggestion
NestedInstallerFiles:
- RelativeFilePath: HTTP_Downloader.exe
  PortableCommandAlias: http-downloader
```

**Comment by stephengillie** [comment]
I'm not clear on the [Policy 2.9](https://learn.microsoft.com/en-us/windows/package-manager/package/windows-package-manager-policies#29-excessive-profanity-and-inappropriate-content) issue, and will need to validate, once the commands are sorted.

**Review by Trenly — APPROVED** [review]

**Comment by stephengillie** [comment]
Policy 2.9 check:
image
image
Everything seems normal. Not seeing a cause for the poilcy check.  

Verify while here:
image
image

**Pull request merged**